### PR TITLE
Create clade hierarchy file

### PR DESCRIPTION
**Pull request by corneliusroemer**
## Description of proposed changes

This PR adds a machine and human readable `clade_hierarchy` file that lists for each clade the parent and also the WHO name (if any).

### CODE REVIEW

**Comment by emmahodcroft** [comment]
I think this is a very nice simple file. The main question for me would be if `defaults` is the right place for it - this is mostly a place where we keep files used in the default configuration for runs, and this file isn't "used" as such. However, it may be there's not really a better place, but would be interested if anyone had alternative suggestions for location.

**Comment by jameshadfield** [comment]
:+1: thanks @corneliusroemer. Also cc @victorlin @j23414 as it would be great to have this displayed in the "resources" section of the documentation.

**Comment by emmahodcroft** [comment]
I think from the comments above we're good Cornelius!

**Review by emmahodcroft — APPROVED** [review]

**Comment by j23414** [comment]
I feel like there should be a way to automatically check for broken links, maybe like this [gh-action/broken-link-check](https://github.com/marketplace/actions/broken-link-check)? Or some manual command?

For linking to docs and after we decide `defaults` or `source-data`, I assume we're updating the [orientation-files.md](https://github.com/nextstrain/ncov/blob/master/docs/src/analysis/orientation-files.md), under "Workflow configuration files...".

**Review by j23414 — APPROVED** [review]

File: defaults/clade_hierarchy.tsv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+21D	20A	Eta
+21E	20B	Theta
+21F	20C	Iota
+21G	20D	Lambda
+21H	20A	Mu
+21I	21A	Delta
+21J	21A	Delta
+21K	21M	Omicron
+21L	21M	Omicron
+21M 20B Omicron
```
Comment (victorlin) [inline defaults/clade_hierarchy.tsv:25]:
```suggestion
21M	20B	Omicron
```

**Comment by emmahodcroft** [comment]
Shall we merge this in?

**Pull request merged**